<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>830</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/3.배경.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../page3resource.qrc">:/back/3.배경.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>742</y>
     <width>46</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/홈버튼 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>754</y>
     <width>33</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/뒤로가기 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tip_btn">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>749</y>
     <width>43</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/팁버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="legback_btn">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>184</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/허벅지 후면.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="all_btn">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>184</y>
     <width>120</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/하체 전체.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="legfront_btn">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>184</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/허벅지 앞면.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="legout_btn">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>376</y>
     <width>120</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/외측장근.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="legin_btn">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>376</y>
     <width>120</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/내전근.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../page3resource.qrc"/>
 </resources>
 <connections/>
</ui>
